### PR TITLE
enable ESM version of AMP runtime

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

File: platform/lib/routers/growPages.js [hunk 1]
```diff
@@ -243,13 +243,7 @@ growPages.get(/^(.*\/)?([^\/\.]+|.+\.html|.*\/|$)$/, async (req, res, next) => {
...
     const optimize = req.query.optimize !== 'false';
     if (optimize) {
-      const experimentEsm = !!req.query.esm || false;
-      const optimizeHeroImages =
-        req.query.hero === undefined ? true : !!req.query.hero;
-      const params = {
-        experimentEsm,
-        optimizeHeroImages,
-      };
+      const params = {};
```
Comment (patrickkettner) [inline platform/lib/routers/growPages.js:246]: this isn't being used, its just being passed to `transformHtml`,  which is [passing it untouched](https://github.com/ampproject/amp-toolbox/blob/94956207cd0de658da9553424c4881481e96a612/packages/optimizer/lib/DomTransformer.js#L125-L127) to `transformTree`, which has a [default of `{}`](https://github.com/ampproject/amp-toolbox/blob/94956207cd0de658da9553424c4881481e96a612/packages/optimizer/lib/DomTransformer.js#L136).

Is there a reason this needs to be here?

File: .husky/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+_
```
Comment (patrickkettner) [inline .husky/.gitignore:1]: ?

File: platform/lib/routers/growPages.js (hunk 1 above)
Comment (sebastianbenz) [reply platform/lib/routers/growPages.js:246]: It's used here `params.maxHeroImageCount = 5;`

```
      const params = {};
      // Enable blurred placeholders and render all 5 stage images of the homepage
      if (req.path === '/') {
        // Disabled until we know why it fails after build
        // params.blurredPlaceholders = true;
        params.maxHeroImageCount = 5;
      }
      renderedTemplate = await optimizer.transformHtml(
        renderedTemplate,
        params

```

**Review by patrickkettner — APPROVED** [review]

File: platform/lib/routers/growPages.js (hunk 1 above)
Comment (patrickkettner) [reply platform/lib/routers/growPages.js:246]: oh pft. totally missed it. lgtm!

**Pull request merged**